lib/src/cmcd/CmcdV2Object.ts

```diff
@@ -0,0 +1,354 @@
+import type { CmcdCustomKey } from './CmcdCustomKey';
+import type { CmcdValue } from './CmcdValue';
+import type { CmcdStreamingFormat } from './CmcdStreamingFormat';
+import type { CmcdStreamType } from './CmcdStreamType';
+import type { CmcdPlayerState } from './CmcdPlayerState';
+
+export const CMCD_V2_COMMON_KEYS = [
+	'br', 
+	'ab', 
+	'bl', 
+	'tbl', 
+	'bs', 
+	'cdn', 
+	'cid', 
+	'ltc', 
+	'mtp', 
+	'pr', 
+	'sf', 
+	'sid', 
+	'bg', 
+	'sta', 
+	'st', 
+	'ts', 
+	'tpb', 
+	'tb', 
+	'lb', 
+	'tab', 
+	'lab', 
+	'pt', 
+	'ec', 
+	'msd', 
+	'v',
+] as const;
+
+/**
+ * Common Media Client Data (CMCD) is a standardized set of HTTP request header fields and query string parameters.
+ *
+ * @group CMCD
+ *
+ * @beta
+ */
+export type CmcdV2Object = {
+
+	/**
+	 * Custom key names may be used, but they MUST carry a hyphenated prefix to ensure that there will not be a namespace collision
+	 * with future revisions to this specification. Clients SHOULD use a reverse-DNS syntax when defining their own prefix.
+	 */
+	[index: CmcdCustomKey]: CmcdValue;
+
+	////////////////////
+	// CMCD V2 Object //
+	////////////////////
+
+	/**
+	 * Encoded bitrate
+	 *
+	 * The encoded bitrate of the audio or video object being requested. This may not be known precisely by the player; however,
+	 * it MAY be estimated based upon playlist/manifest declarations. If the playlist declares both peak and average bitrate values,
+	 * the peak value should be transmitted.
+	 *
+	 * Integer kbps
+	*/
+	br?: number;
+
+	/**
+	 * Aggregate encoded bitrate
+	 *
+	 * The aggregate encoded bitrate of the complete media object including all object types. This SHOULD be derived from 
+	 * playlist/manifest declarations, or it MAY be estimated by the player. If the playlist declares both peak and average bitrate values, 
+	 * the peak value MUST be transmitted. This value MUST NOT be sent for objects which do not have an object type of ‘a’, ‘v’, ‘av’ or ‘o’. 
+	 * This value MUST NOT be sent if the encoded bitrate is known.
+	 * 
+	 * Integer kbps
```

Comment:
I would suggest expanding on what b in kbps means (for clarity), to avoid confusion, e.g.

```suggestion
	 * Integer in kilobits per second
```

Reply:
We will address this once the CMCD version 2 spec is finalized.